### ISSUE TITLE
DOC: Description of table syntax incorrect

### ISSUE THREAD

**Issue by mcarden** [sphinx-doc/sphinx#3863]
Subject: The Sphinx documentation at http://www.sphinx-doc.org/en/stable/rest.html contains a description of the Table syntax which appears to be incorrect.

### Problem
The explanation in the 'Tables' section of sphinx/doc/rest.rst contains the text "...the first column cannot contain multiple lines..." which seems not to be correct. The first column (the A column in the example) does contain multiple lines as perhaps any table should. It seems that what is intended is to state that the first ROW (or more correctly, the header ROW) cannot contain multiple lines.

#### Procedure to reproduce the problem
```
Examine the 'Tables' section of the doc published at http://www.sphinx-doc.org/en/stable/rest.html
```

#### Error logs / results
```
N/A```

#### Expected results
N/A

### Reproducible project / your project
N/A

### Environment info
N/A

**Comment by jfbu** [comment]
Thanks, but no, it is not the first row, 
Dixit [docutils docs](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#simple-tables):

> For this reason, cells in the first column of new rows (not continuation lines) must contain some text; blank cells would lead to a misinterpretation (but see the tip below). Also, this mechanism limits cells in the first column to only one line of text. 

Hence a better formulation could be in Sphinx docs

> Simple tables (ref) are easier to write, but limited: they must contain more than one row, and the first column cells cannot contain multiple lines.

**Comment by jfbu** [comment]
Fixed at 52115854. Thanks for reporting!
